File: app/src/main/java/com/inspiredandroid/linuxcommandbibliotheca/fragments/DatabaseLoadingFragment.java
package com.inspiredandroid.linuxcommandbibliotheca.fragments;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.inspiredandroid.linuxcommandbibliotheca.R;

public class DatabaseLoadingFragment extends Fragment {

    public DatabaseLoadingFragment()
    {
        // Required empty public constructor
    }

    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState)
    {
        return inflater.inflate(R.layout.fragment_database_loading, container, false);
    }

}

File: app/src/main/java/com/inspiredandroid/linuxcommandbibliotheca/fragments/DatabaseLoadingFragment.kt
package com.inspiredandroid.linuxcommandbibliotheca.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.inspiredandroid.linuxcommandbibliotheca.R

/**
 * Created by Simon Schubert
 */
class DatabaseLoadingFragment : Fragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_database_loading, container, false)
    }
}
